src/ImageSharp/Common/Helpers/EnumUtils.cs

```diff
@@ -19,15 +20,14 @@ internal static class EnumUtils
         /// <param name="defaultValue">The default value to return.</param>
         /// <returns>The <typeparamref name="TEnum"/>.</returns>
         public static TEnum Parse<TEnum>(int value, TEnum defaultValue)
-            where TEnum : Enum
+            where TEnum : struct, Enum
         {
-            foreach (TEnum enumValue in Enum.GetValues(typeof(TEnum)))
+            Debug.Assert(Unsafe.SizeOf<TEnum>() == sizeof(int), "Only int-sized enums are supported.");
```

Comment:
```suggestion
            DebugGuard.IsTrue(Unsafe.SizeOf<TEnum>() == sizeof(int), "Only int-sized enums are supported.");
```

We don't use `Debug.Assert`.

Reply:
Fixed.